libmusic/include/lmtypes.h

```diff
@@ -145,22 +159,47 @@ typedef struct Chord {
                 ((c1.__mRootNote == c2.__mRootNote) && (c1.__mQuality < c2.__mQuality)));
     }
 
-    friend std::ostream& operator<<(std::ostream& os, const Chord &c)
+    std::string toHarte() const
     {
-        os << c.__mRootNote << c.__mQuality;
-        if (c.__mBassNote != note_Unknown)
-            os << "/" << c.__mBassNote;
-        return os;
+        std::ostringstream ss;
+
+        ss << __mRootNote;
+        if (__mRootNote != note_Unknown) {
+            ss << ":" << __mQuality;
+        }
+        if (__mBassInterval > 1) {
+            ss << "/" << (int)__mBassInterval;
+        }
+
+        return ss.str();
     }
 
-    std::string toString()
+    std::string toUser() const
     {
         std::ostringstream ss;
+
         ss << __mRootNote << __mQuality;
         if (__mBassNote != note_Unknown)
             ss << "/" << __mBassNote;
+
         return ss.str();
     }
+
+    std::string toString() const
```

Comment:
there's no other user of this method then `operator<<()`, so maybe we can put its code under `operator<<` and avoid this unnecessary nesting?

Reply:
lmclient uses it

Reply:
oops, seem my code indexer sucks :S